### PR TITLE
gen-bundle-image: Add arm64 bundle image creation for linux

### PR DESCRIPTION
**Pull request by praveenkumar**

### CODE REVIEW

File: gen-bundle-image.sh [hunk 1]
```diff
@@ -67,6 +74,7 @@ function sign_bundle_files {
   rm -fr *.sig
   if [[ ${preset} != "okd" ]]; then
      gpg --batch --default-key crc@crc.dev --pinentry-mode=loopback --passphrase-file ${GPG_SECRET_KEY_PASSPHRASE_PATH} --armor --output ${vfkit_bundle_arm64}.sig --detach-sig ${vfkit_bundle_arm64}
+     gpg --batch --default-key crc@crc.dev --pinentry-mode=loopback --passphrase-file ${GPG_SECRET_KEY_PASSPHRASE_PATH} --armor --output ${libvirt_bundle_arm64}.sig --detach-sig ${libvirt_bundle_arm64}
```
Comment (anjannath) [inline gen-bundle-image.sh:77]: should this be outside of the `okd` check, since image will be generated also for okd?

File: gen-bundle-image.sh [hunk 2]
```diff
@@ -19,6 +19,7 @@ function set_bundle_variables {
 
    vfkit_bundle=crc${bundlePreset}_vfkit_${version}_amd64.crcbundle
    libvirt_bundle=crc${bundlePreset}_libvirt_${version}_amd64.crcbundle
+   libvirt_bundle_arm64=crc${bundlePreset}_libvirt_${version}_arm64.crcbundle
```
Comment (anjannath) [inline gen-bundle-image.sh:22]: ahh, so we don't want to generate arm64 libvirt bundles only for `okd` by looking at the changes, down.. so maybe this should be inside the `okd` check above at L17

Comment (praveenkumar) [reply gen-bundle-image.sh]: @anjannath done.

File: gen-bundle-image.sh (hunk 1 above)
Comment (praveenkumar) [reply gen-bundle-image.sh:77]: No, we are not having arm64 support for okd.

**Review by anjannath — APPROVED** [review]

**Pull request merged**